src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -186,11 +300,342 @@ A guide for using macros can be found at
    {
    "sub-01": {
       "func": {
-        "sub-01_task-cuedSGT_run-1_physio.tsv.gz": "",
-        "sub-01_task-cuedSGT_run-1_echo-1_bold.nii.gz": "",
-        "sub-01_task-cuedSGT_run-1_echo-2_bold.nii.gz": "",
-        "sub-01_task-cuedSGT_run-1_echo-3_bold.nii.gz": "",
+         "sub-01_task-nback_physio.json": "",
+         "sub-01_task-nback_physio.tsv.gz": "",
+         "sub-01_task-nback_physioevents.json": "",
+         "sub-01_task-nback_physioevents.tsv.gz": "",
+         },
+      },
+   }
+) }}
+
+where the decompressed contents of `sub-01_task-nback_physio.tsv.gz`
+are:
+
+```TSV linenums=1
```

Comment:
```suggestion
```TSV
```

Reply:
From the rendered HTML I think that mkdocs does not deal well with linenums

Reply:
figured out :)